### ISSUE TITLE
Create rule 'post-composing'

### ISSUE THREAD

**Issue by chris263** [solgenomics/sgn#4585]
Expected Behavior 
--------------------------------------------------------------------------

Create a rule to allow only selected people to post compose traits.

For Bugs:
---------

**Comment by chris263** [comment]
The idea is to allow only some selected submitters to post compose traits. Then, they will be submitter + post-composing.

**Comment by nmenda** [comment]
@chris263 do we need a new user role? 'post composer' ?

**Comment by chris263** [comment]
Yes, we need a post compose role. It should be the same as submitter, however we should allow only pot composer to create new traits. Then, submitter still sending data but not creating traits.

**Comment by lukasmueller** [comment]
working on this now. Adding a postcomposing role that will be the only one allowed to postcompose traits in addition to curator.
